modules/web/web-vhost/src/main/java/com/enonic/xp/web/vhost/impl/mapping/VirtualHostMapping.java

```diff
@@ -19,10 +20,18 @@ public final class VirtualHostMapping
 
     private final int order;
 
+    private final Map<String, String> context;
```

Comment:
How you guarantee it is immutable?

Reply:
https://github.com/enonic/xp/pull/10267/files#diff-de7ef834b70f3087be55fce7e3e1ce37a58ad5976c4b95f0dfc129d2684a6fffR49